### ISSUE TITLE
New marketplace Ireland - VCS eligibility

### ISSUE THREAD

**Issue by Dragonsangel** [amzn/selling-partner-api-models#4415]
As per the [Announcement](https://developer-docs.amazon.com/sp-api/changelog/ireland-ie-marketplace-launch-for-seller-central) on the 11th December, the Ireland marketplace has been launched. Products and orders can be retrieved from the marketplace.

However, the Ireland marketplace is also within the EU and therewith the question if VAT-Calculations are also generated and invoice uploads required for the orders originating from the marketplace?

Within the [documentation for manual invoice upload](https://developer-docs.amazon.com/sp-api/docs/option-3-upload-invoices-through-sp-api-or-seller-central), both Ireland and Belgium are not in the list, yet we know that Belgium is enrolled in the VCS. However, in the [Option 2 variation](https://developer-docs.amazon.com/sp-api/docs/option-2-use-amazon-tax-calculation-data-to-create-data-invoices-and-upload-with-sp-api#step-3-upload-the-invoices-to-amazon-using-the-feeds-api), Belgium is in the list, but not Ireland.
Could we have a "Marketplaces enrolled in VCS" page in the SP-API documentation, that gets updated as new marketplaces are launched?

tl;dr:

1. Is Ireland enrolled in VCS, requiring invoice uploads?
2. Could we have a dedicated "VCS enrolled Marketplaces"-Page in the documentation?

Thanks,
Bernd

**Comment by Dragonsangel** [comment]
The documentation has not been updated or created, nor has the original question been answered, so this should please remain open.

**Comment by Dragonsangel** [comment]
For those that find this issue and asking the same regarding Ireland and VCS, herewith a response from Amazon Developer Support:

> Currently, the supported marketplaces for VCS through the API are Spain, UK, France, Netherlands, Germany, Italy, Switzerland, and Poland. 
> Ireland is currently not supported.

However, this list is incomplete, since we have VCS-Data for Orders from the Belgium and Swedish marketplaces, meaning they are also supported for VCS. 
Switzerland is also [not a known marketplace](https://developer-docs.amazon.com/sp-api/docs/marketplace-ids).

**Comment by Dragonsangel** [comment]
The documentation does not reflect the requested information and therefore the issue should remain open

**Comment by Dragonsangel** [comment]
Still outstanding, no close comment

**Comment by jawheral** [comment]
Hi @Dragonsangel,

The VAT Calculation Service (VCS) is available in the Ireland (IE) marketplace and will support VAT invoicing automation as in other EU marketplaces.

I'm in contact with our teams to update our documentation regarding marketplace availability.

Thank you for the feedback.

Best,
Jawher
Selling Partner Developer Services

**Comment by Dragonsangel** [comment]
As mentioned above, the documentation update is still outstanding and hence no auto close.

**Comment by Dragonsangel** [comment]
As mentioned above, the documentation update is still outstanding and hence no auto close.

**Comment by Dragonsangel** [comment]
As mentioned above, the documentation update is still outstanding and hence no auto close.

**Comment by Dragonsangel** [comment]
The documentation update is still outstanding and hence no auto close.

**Comment by jawheral** [comment]
Hi @Dragonsangel,

VCS related pages are now displaying the up-to-date list of eligible marketplaces. 
Thank you for the feedback.

Best,
Jawher
Selling Partner Developer Services
